src/Generator/Library.cs

```diff
@@ -175,10 +175,15 @@ public static void SetClassBindName(this ASTContext context, string className, s
                 @class.Name = name;
         }
 
-        public static void SetClassAsValueType(this ASTContext context, string className)
+        public static void SetClassAsValueType(this ASTContext context, string className, Driver driver)
```

Comment:
I changed this to behave similar to CS_VALUE_TYPE macro and not to make value type for specific classes when in CLI mode.
@tritao Please review this part and if it is acceptable now (As the AST is changed only in this function and the macro) please merge it.